api/src/main/java/org/apache/iceberg/Snapshot.java

```diff
@@ -162,6 +162,16 @@ default Iterable<DeleteFile> removedDeleteFiles(FileIO io) {
    */
   String manifestListLocation();
```

Comment:
Is `String manifestListLocation()` same as what is stored in ManifestListFile? If so, do we still need it as a separate field?

Reply:
@rdblue @RussellSpitzer what do you think?

Reply:
There are dozens of calls to this method today (tests etc), so we'll likely need to keep it for a while.

Reply:
Yeah, I agree. We need to keep this because it is part of the public API and used in many cases.